### ISSUE TITLE
Appends existing data to .gitslave and .gitignore

### ISSUE THREAD

**Issue by reggi** [joelpurra/gitslave#4]
I would love `.gitslave` to be a little smarter about saving this data when it's not needed, especially because it commits for you.

**Comment by joelpurra** [comment]
@reggi: how do you mean "when it's not needed"? Do you mean that you hand-edit the `.gitslave` and `.gitignore` then `gits attach` or something?

**Comment by reggi** [comment]
Nope. Here's an explication, If I run the following twice

`gits attach https://github.com/joelpurra/gitslave.git gitslave`

I'll end up with a `.gitignore` that has.

```
/gitslave/
/gitslave/
```

and a `.gitslave` with

```
https://github.com/joelpurra/gitslave.git gitslave
https://github.com/joelpurra/gitslave.git gitslave
```

Behavior should be, if line exists do nothing, if not add it.

**Comment by joelpurra** [comment]
@reggi: you're right. This should be reported upstream, but please check if it has already been reported first.

I'll leave this issue open. Pull requests are welcome!
